src/Session.cpp

```diff
@@ -684,6 +684,14 @@ bool SESSION::CSession::PrepareStream(CStream* stream, uint64_t startPts)
                                                    DRM::DecrypterCapabilites::SSD_SECURE_PATH);
   }
 
+#ifdef TARGET_WEBOS
+  if (adp->GetStreamType() == StreamType::AUDIO)
+  {
+    m_reprChooser->SetSecureSession(session && session->capabilities.flags &
+                                                   DRM::DecrypterCapabilites::SSD_SECURE_PATH);
+  }
+#endif
+
```

Comment:
this can be fully removed, maybe its not so clear but the only reason that "representation choosers" need to know this is for video purpose not audio,
this is related to setting "maximum resolution for drm videos" its a workaround for RPI & similars, low-powered CPUs cannot process correctly high resolutions

i would like in a future full remove this workaround setting, but idk...